pepper-apis/studybuilder-cli/studies/lms/emails/somatic-results-invitation.html

```diff
@@ -0,0 +1,121 @@
+<html>
+<meta name="format-detection" content="telephone=no,date=no,address=no,email=no,url=no" />
```

Comment:
HTML formatting is copied from existing lms another email template. Looks good but may be the html formatting can be simplified. GiCharkviani : Time permitting may be you can take a look at this once All other pe-cgs stuff is completed.